ChangeLog.d/fix-pkgconfig-static-linking-windows.txt

```diff
@@ -0,0 +1,2 @@
+Bugfix:
+   * Fix pkgconfig files on Windows: the bcrypt library was missing when linking statically.
```

Comment:
Sorry, one more thing: please limit the line length to 80 columns (preferably 79).